src/test/java/com/thomashan/coup/turn/state/WaitingChallengeMainActionStateTestCase.java

```diff
@@ -0,0 +1,11 @@
+package com.thomashan.coup.turn.state;
+
+import com.thomashan.coup.action.MainAction;
+
+import static java.util.Collections.emptyList;
+
+public class WaitingChallengeMainActionStateTestCase extends TurnStateTestCase {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This class name ends with Test but contains no test cases](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)